src/components/Activities/Activity1/Game1/components/CipherGameRound.tsx

```diff
@@ -71,13 +71,33 @@ function CipherGameRound(props : CipherGameRoundProps): JSX.Element {
     }, 1000);
   };
   // ------------------------
+  const vowelize = (word : string) : string => { // sets the first character of a word to the nearest vowel
+    let char = word.toUpperCase().charCodeAt(0);
+
+    const vowels = ['A', 'E', 'I', 'O', 'U'];
+    for (let i = 0; i < vowels.length; i++) { // iterate and set 1st letter to a vowel
+      const vowelCode = vowels[i].charCodeAt(0);
+      if (char <= vowelCode || i === vowels.length-1) {
+        char = vowelCode;
+        break;
+      }
+    }
```

Comment:
if you see for loops, try to think if theres a functional way to do it since it usu ends up with cleaner + less + easier to read code

```suggestion
       const closestVowel = vowels.find((vowel) => charCode <= vowel.charCodeAt(0)) || 'U';
```

Reply:
ok!